server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -241,6 +241,9 @@ export function configureWebSocketServices(
 		// Map from client Ids to connection time.
 		const connectionTimeMap = new Map<string, number>();
 
+		let connectDocumentComplete: boolean = false;
```

Comment:
Will there be an instance of `connectDocumentComplete` for each connection? That seems to be the case given line 234 above (`webSocketServer.on("connection") ...`, but I wanted to double check just in case - since we reuse this on disconnect, line 852 below, my concern (probably not valid) is whether we could be looking into `connectDocumentComplete` that refers to a different connection/disconnect event.